src/streamer.rs

```diff
@@ -224,19 +224,21 @@ fn repair_window(
     //if times flips from all 1s 7 -> 8, 15 -> 16, we retry otherwise return Ok
     if *times & (*times - 1) != 0 {
         trace!("repair_window counter {} {} {}", *times, consumed, received);
-        return Ok(());
```

Comment:
If you're going to remove this early return, should we remove the trace as well? Either way the comment is wrong.

Removing exponential backoff seems like a dangerous move, but I'm okay with it, since it didn't break any of @aeyakovenko's tests.